release/llm_tests/batch/llm_4xl4.yaml

```diff
@@ -0,0 +1,15 @@
+cloud_id: {{env["ANYSCALE_CLOUD_ID"]}}
+region: us-west-2
+
+head_node_type:
+    name: head_node
+    instance_type: m5.2xlarge
+    resources:
+      cpu: 0
+
+worker_node_types:
```

Comment:
can we use the auto worker node types similar to serve and use only one yaml? Trying to not create extra yamls if it's not necessary.

Reply:
But auto select worker is not working well in my case so I don't want to use it rn.